src/navs/index.js

```diff
@@ -2022,6 +2022,10 @@ export const docsMenu = {
                             name: 'Data deletion',
                             url: '/docs/privacy/data-deletion',
                         },
+                        {
+                            name: 'Ad blockers',
```

Comment:
I feel like people will see this and think "How to handle ad blockers." Options:

1. Change this (and file name) to `Tracking endpoints`
2. Add link to reverse proxy. Maybe this is a bit troll though: "hey here's are tracking endpoints but also here is a way to get around blocking them lol"

Reply:
#1 is much more subtle as MENU option, what if we split the difference and change the nav but keep the title/H1 so we still rank for "PostHog ad blocker" as Raquel hoped?

Reply:
Ahh, I see. I guess I'm not fully convinced people Googling this want to block PostHog. For example, this is first non-PostHog link:

![CleanShot 2024-12-19 at 14 03 29](https://github.com/user-attachments/assets/558bf4ed-2b4f-4661-bf67-2feb8cb75198)

Fine with me as is tbh